desci-server/src/services/PublishPackage.ts

```diff
@@ -47,29 +47,33 @@ class PublishPackageService {
       this.logger.error('process.env.ISOLATED_MEDIA_SERVER_URL is not defined');
       return null;
     }
+    // debugger;
     const title = manifest.title;
-    const dpid = manifest.dpid.id;
+    const demoMode = manifest?.dpid?.id === undefined;
+    const dpid = !demoMode ? manifest?.dpid?.id : 'UNPUBLISHED_DEMO';
     if (dpid === undefined) {
       this.logger.warn({ dpid, nodeId: node.id }, 'Failed generating a publish package for node, dpid is undefined');
       throw new Error('DPID is undefined');
     }
 
     const license = PublishPackageService.extractManuscriptLicense(manifest, pdfCid);
-    const publishTime = await publishServices.retrieveBlockTimeByManifestCid(node.uuid, manifestCid);
+    const publishTime = demoMode
+      ? Date.now().toString().slice(0, 8)
+      : await publishServices.retrieveBlockTimeByManifestCid(node.uuid, manifestCid);
     const publishDate = PublishPackageService.convertUnixTimestampToDate(publishTime);
     const authors = manifest.authors?.map((author) => author.name);
 
-    const attestations = await attestationService.getAllNodeAttestations(dpid);
+    const attestations = await attestationService.getAllNodeAttestations(node.uuid);
 
     const openCodeAttestation = attestations.find((a) => a.attestationId === 15);
```

Comment:
this won't work across environments, the id of these attestations is different in dev vs prod

Reply:
perhaps do a string match on the name